ext/truth/java/androidx/test/ext/truth/os/SparseBooleanArraySubject.java

```diff
@@ -0,0 +1,118 @@
+package androidx.test.ext.truth.os;
+
+import android.util.SparseBooleanArray;
+
+import com.google.common.truth.ExpectFailure;
+import com.google.common.truth.Fact;
+import com.google.common.truth.FailureMetadata;
+import com.google.common.truth.Subject;
+import com.google.common.truth.Truth;
+
+/**
+ * Subject for making assertions about {@link SparseBooleanArray}s.
+ */
+public final class SparseBooleanArraySubject extends Subject {
+
+    private final SparseBooleanArray actual;
+
+    private static Factory<SparseBooleanArraySubject, SparseBooleanArray> sparseBooleanArrays() {
+        return SparseBooleanArraySubject::new;
+    }
+
+    SparseBooleanArraySubject(FailureMetadata failureMetadata, SparseBooleanArray subject) {
+        super(failureMetadata, subject);
+        this.actual = subject;
+    }
+
+    /**
+     * Assert that a {@link SparseBooleanArray} for the given key returns the value true
+     */
+    public SparseBooleanArraySubject keyIsTrue(int key) {
```

Comment:
I must admit I liked the original naming best (`keyIsTrue`) although I understand that it could be misunderstood. I'll change it to `isTrueAt` and `isFalseAt` as I tend to use prefer shorter function-names.

Reply:
After reading the usage of this change in the tests I think changing it to `hasTrueValueAt` like @TWiStErRob suggested makes the most sense. I think it reads better:

`assertThat(subject).hasTrueValueAt(42);` // Reads to me: subject has true value at

vs.

`assertThat(subject).isTrueAt(42);` // Reads to me: subject is true at

WDT?